README.md

```diff
@@ -265,6 +267,84 @@ const emailParams = new EmailParams()
 mailersend.send(emailParams);
 ```
 
+### Send bulk emails
+```js
+const Recipient = require("mailersend").Recipient;
+const EmailParams = require("mailersend").EmailParams;
+const BulkEmails = require("mailersend").BulkEmails;
+const MailerSend = require("mailersend");
+
+const mailersend = new MailerSend({
+  api_key: "key",
+});
+
+const bulkEmails = new BulkEmails();
+
+const recipients = [
+  new Recipient("your@client.com", "Your Client")
+];
+
+const emailParams = new EmailParams()
+  .setFrom("your@domain.com")
+  .setFromName("Your Name")
+  .setRecipients(recipients)
+  .setSubject("Subject")
+  .setHtml("This is the HTML content")
+  .setText("This is the text content");
+
+
+bulkEmails.addEmail(emailParams)
+bulkEmails.addEmails([
+  emailParams,
+  emailParams
+])
+
+mailersend.sendBulk(bulkEmails)
+  .then((response) => response.json())
+  .then((data) => {
+    console.log(data);
+  });
+```
+
+### Get bulk request status
+```js
+const Recipient = require("mailersend").Recipient;
+const EmailParams = require("mailersend").EmailParams;
+const BulkEmails = require("mailersend").BulkEmails;
+const MailerSend = require("mailersend");
+
+const mailersend = new MailerSend({
+  api_key: "key",
+});
+
+const bulkEmails = new BulkEmails();
+
+const recipients = [
+  new Recipient("your@client.com", "Your Client")
+];
+
+const emailParams = new EmailParams()
+  .setFrom("your@domain.com")
+  .setFromName("Your Name")
+  .setRecipients(recipients)
+  .setSubject("Subject")
+  .setHtml("This is the HTML content")
+  .setText("This is the text content");
+
+
+bulkEmails.addEmail(emailParams)
+
+mailersend.sendBulk(bulkEmails)
```

Comment:
Maybe we can show this example without calling the `sendBulk` method? Because we just need to pass the `bulk_email_id` to the `getBulkEmailStatus`. In that way, it'd be clear that someone can just store the `bulk_email_id` from a previous call and later check the status based on the `bulk_email_id`. From this example, it seems like the status can be checked only right after the `sendBulk` call.